src/background.js

```diff
@@ -4,16 +4,18 @@
 
 const TRACKING_ID = 'UA-35433268-79';
 
-const KINTO_SERVER = 'https://testpilot.settings.services.mozilla.com/v1';
+//const KINTO_SERVER = 'https://testpilot.settings.services.mozilla.com/v1';
+//const KINTO_SERVER = 'http://localhost:8888/v1';
+const KINTO_SERVER = 'http://kinto-notes5.lcip.org/v1';
 // XXX: Read this from Kinto fxa-params
-const FXA_CLIENT_ID = 'a3dbd8c5a6fd93e2';
-const FXA_OAUTH_SERVER = 'https://oauth.accounts.firefox.com/v1';
-const FXA_PROFILE_SERVER = 'https://profile.accounts.firefox.com/v1';
+//const FXA_CLIENT_ID = 'b7d74070a481bc11';
```

Comment:
I think ESLint only failed due to lack of a space after the `//` comments on L7, L8, and L11.